homeassistant/components/recorder/models.py

```diff
@@ -221,7 +219,7 @@ def to_native(self, validate_entity_id=True):
                 process_timestamp(self.last_updated),
                 # Join the events table on event_id to get the context instead
                 # as it will always be there for state_changed events
-                context=Context(id=None),
+                context=Context(id=None),  # type: ignore[arg-type]
```

Comment:
This is only visible now that the function is typed. There are a few of these that need to be adjusted in the codebase but the tests will break (not a problem in production) so I didn't want to open up that in this PR as it had the potential to be larger than this PR itself.